portfolio.css

```diff
@@ -0,0 +1,53 @@
+body{
+  background-color: lightgray;
+  margin: 0;
+}
+
+.wrapper {
+  max-width: 620px;
+  display: grid;
+  grid-template-columns: repeat (2, 1fr);
+  grid-template-rows: repeat (5, clearçç1fr);
+  grid-row-gap: 10px;
+
+}
+
+
+.content {
+  margin: 0px 7px;
+}
+
+.heading {
+  margin: 0px auto 20px;
+  width: 600px;
+  height: 250px;
+  border: 4px solid black;
+  background-color: white;
+}
+
+h1,h2 {
+  text-align: center;
+}
+
+h1 {
+  font-family: 'Graduate' , sans-serif;
+  font-size: 80px;
+}
+
+h2 {
+  font-family: 'Six+Caps' , cursive;
+}
+
+
+footer{
```

Comment:
Have you thought about using flexbox to sort the alignment issues?

Reply:
will do it on a different branch.